hat/assets/js/apps/Iaso/domains/orgUnits/components/OrgUnitsFilters.tsx

```diff
@@ -258,7 +260,7 @@ export const OrgUnitFilters: FunctionComponent<Props> = ({
                     label={MESSAGES.search}
                     blockForbiddenChars
                     onEnterPressed={onSearch}
-                    onErrorChange={setTextSearchError}
+                    onErrorChange={(hasError) => setTextSearchError(hasError)}
```

Comment:
This is just to make the TS compiler happy isn't it?😁

Reply:
yep